sass/globals/_colors.sass

```diff
@@ -0,0 +1,19 @@
+$black: #000
+$white: #fff
+
+// Greys
+$base-grey: #7f7f7f !default
+// light greys
+$grey-1: lighten($base-grey, 45%)
+$grey-2: lighten($base-grey, 40%)
+$grey-3: lighten($base-grey, 35%)
+$grey-4: lighten($base-grey, 30%)
+$grey-5: lighten($base-grey, 20%)
+$grey-6: lighten($base-grey, 10%)
+// base grey
+$grey-7: $base-grey
```

Comment:
I would assume the lower index would be darker, so `$grey-1` is darkest and `$grey-11` is lightest, scaling from black to white